ssl/statem/extensions_srvr.c

```diff
@@ -995,7 +998,7 @@ int tls_construct_stoc_next_proto_neg(SSL *s, WPACKET *pkt,
 
     s->s3->npn_seen = 0;
```

Comment:
Do we want to try to follow up and replace uses of things like npn_seen with the extflags bit?